app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+
+  acts_as_tree order: "created_at ASC"
```

Comment:
để asc thôi đc ko, tại sao lại dùng chữ in hoa nhỉ

Reply:
Em chưa thử để lowercase vì từ lúc em học em vẫn để uppercase theo nguồn này ạ.
https://www.w3schools.com/sql/sql_orderby.asp

Reply:
Cả nguồn của mysql cũng để vậy.
https://dev.mysql.com/doc/refman/8.0/en/order-by-optimization.html

Reply:
Được anh ạ vì em vừa thử bằng query hehe...

Reply:
Và em thấy các người đều để in hoa. Em nghĩ để dễ nhận biết ạ!
Source: https://dev.mysql.com/doc/refman/8.0/en/order-by-optimization.html